client/blocks/plugins-update-manager/index.tsx

```diff
@@ -2,7 +2,7 @@ import { Button } from '@wordpress/components';
 import { plus } from '@wordpress/icons';
 import MainComponent from 'calypso/components/main';
 import NavigationHeader from 'calypso/components/navigation-header';
-import { useScheduleUpdatesQuery } from 'calypso/data/plugins/use-schedule-updates-query';
+import { useScheduleUpdatesQuery } from 'calypso/data/plugins/use-update-schedules-query';
```

Comment:
I think `useScheduleUpdatesQuery` still needs updating

Reply:
Ups, good catch. I have updated it.

Thanks!